.circleci/config.yml

```diff
@@ -209,6 +219,18 @@ commands:
             - vms/
             - wasi-sdk/
             - .jsvu/
+      - when:
+          condition:
+            and:
+              - equal: [ "main", << pipeline.git.branch >> ]
+              - equal: [ "https://github.com/emscripten-core/emscripten", << pipeline.project.git_url >> ]
```

Comment:
Maybe worth comment here?  Why this condition?

Reply:
I originally had it in there to avoid the possibility of having a cache update from a forked PR from the main branch of another repo. But I just tested and I'm not sure it's actually necessary, the branch always appears as "pull/12345". So I could just take it out.